scripts/rff/projected_weather_variability/README.md

```diff
@@ -0,0 +1,31 @@
+# Importing Forecasted Raster of RFF Weather Variability Data
```

Comment:
Could you please confirm if you've uploaded these raw source data to GCS?

Reply:
Yes, followed up in a different email thread.